MANIFEST

```diff
@@ -10,4 +10,5 @@ pyttsx3\drivers\_espeak.py
 pyttsx3\drivers\dummy.py
 pyttsx3\drivers\espeak.py
 pyttsx3\drivers\nsss.py
+pyttsx3\drivers\avsynth.py
```

Comment:
Please keep the list sorted to make it easy to find missing filenames and nearly impossible to add duplicates.